### PR TITLE
Add /api/info endpoint

### PR DESCRIPTION
**Pull request by bobo333**
The endpoint returns the following library information

- libraryExpireDays
- dataExpireDays
- apiVersion

@kzalewski I'm not the most experienced in java, so if there are changes you'd like please let me know. I've tested it successfully locally.

Here's an example of the output (json-formatted by my browser)
image

### CODE REVIEW

File: src/main/java/gov/nysenate/ams/controller/LibraryInfoController.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException
+    {
+        processRequest(request, response);
+    }
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException
```
Comment (bobo333) [inline src/main/java/gov/nysenate/ams/controller/LibraryInfoController.java:37]: initially this was only going to support `GET` but the `BaseApiController` requires both `doGet` and `doPost`, so I had them both do the same thing. Open to suggestions if there is a preferred approach

**Comment by bobo333** [comment]
@kzalewski gentle reminder to please have a look at this when you get a chance. I'm happy to contact someone else as well if there is a more appropriate person.

**Comment by kzalewski** [comment]
Hi bobo333.  I thought we already had an "info" endpoint for the AMS library.  Whenever I upgrade the library (after receiving the latest monthly release from USPS), I check it by navigating to /usps-ams/about.jsp.  I receive a page that displays the library version, the days until the library expires, and the days until the data expires.

**Comment by bobo333** [comment]
hi @kzalewski my mistake, I completely missed that. Would you accept a PR adding documentation for that endpoint?

Either way, I'm closing this one, thank you for the review

**Pull request closed without merging**